### ISSUE TITLE
Failed to build using CocoaPods

### ISSUE THREAD

**Issue by ryanzzff** [maxep/MXSegmentedPager#125]
MXSegmentedPagerController:
`MXPageSegueDelegate` is missing...

```
@interface MXSegmentedPagerController () <MXPageSegueDelegate>
```

MXPagerView (0.1.3) works fine
MXPagerView (0.1.4) is broken

**Comment by JozefLipovsky** [comment]
Since MXPagerView (0.1.4) `MXPageSegueDelegate` was renamed and updated to `MXPageSegueSource`

**Comment by maxep** [comment]
Yes, I try to fix it but cocoapods is [out](https://github.com/CocoaPods/CocoaPods/issues/6098). I will fix it asap

**Comment by JozefLipovsky** [comment]
Just added new pull request, quickly tested on Obj-C Example app.

**Comment by nharbo** [comment]
I have this problem too.. How to fix it? :)

**Comment by ryanzzff** [comment]
Before @maxep fix the problem, I think you can just replace `MXPageSegueDelegate` to `MXPagerViewDelegate` as Xcode suggest.

**Comment by maxep** [comment]
Fixed in 3.2.0
